src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -217,7 +218,7 @@ const Comments = ({
               <li key={comment.id}>
                 {commentDetail}
 
-                <p>{comment.content}</p>
+                <p data-testid="comment-content">{comment.content}</p>
```

Comment:
I would add the same data-testid on to line 201 too, so we can use it to select the content of resolved comments as well.